packages/storybook/src/stories/grid-columns.stories.tsx

```diff
@@ -363,3 +364,31 @@ export const SingleSelectColumnType = () => {
     </div>
   );
 };
+export function FlexLayoutGrid() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 15,
+    maxColumns: 6,
+  });
+
+  const tmpData = JSON.parse(JSON.stringify(data));
+  tmpData.columns.forEach((item) => {
+    item.flex = 1;
+  });
+  return (
+    <div style={{ height: '400px', width: '400px' }}>
+      <div style={{ display: 'flex', height: '100%' }}>
+        <div style={{ flexGrow: 1 }}>
+          <XGrid
+            autoHeight
+            showCellRightBorder
+            showColumnRightBorder
+            disableExtendRowFullWidth
+            rows={tmpData.rows}
+            columns={tmpData.columns}
+          />
+        </div>
+      </div>
+    </div>
+  );
+}
```

Comment:
We can already reproduce the issue in the docs. What's the use case for the story?

```suggestion
```

Reply:
working on flex columns, and see if it works. Where is #2011 in the doc?

Reply:
Ok, could we make it a Snap story in this case? This could complement the function test to make sure the rendering with sub-pixels is still correct